src/FeedContext/Google/Shopping/ProductItemContext.php

```diff
@@ -177,6 +178,24 @@ private function getLink(LocaleInterface $locale, ProductTranslationInterface $t
         );
     }
 
+    private function getVariantImageLink(ProductImagesAwareInterface $imagesAware): ?string {
```

Comment:
```suggestion
    private function getVariantImageLink(ImagesAwareInterface $imagesAware): ?string {
```

Reply:
It wont work, because ProductVariantInterface extends ProductImagesAwareInterface, not ImagesAwareInterface.

Reply:
Okay :)